src/runtime/entitlements-manager.js

```diff
@@ -91,11 +91,12 @@ export class EntitlementsManager {
 
   /**
    * @param {string} raw
+   * @param {?boolean|undefined=} opt_isReadyToPay
```

Comment:
Ditto. Either `undefined` or `=` should be used. Since you use `opt_` - you probably want to use `=` - it means `undefined` and optional.

Reply:
done